docs/index.md

```diff
@@ -115,7 +115,7 @@ Within your existing `didFinishLaunchingWithOptions` method, add the following t
 ```
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
   // Add me --- \/
-  [FIRApp configure];
+  if ([FIRApp defaultApp] == nil) { [FIRApp configure]; }
```

Comment:
This "check for null / only configure if defaultApp is null" has been the source of some debate, I'm not sure it is strictly required and I should probably remove it from my demo script.

Can you test if it works just fine as currently documented, that is just a straight `[FIRApp configure];` call?

Reply:
@mikehardy 
it works with `[FIRApp configure];`
so I will change this as currently documented!